script.js

```diff
@@ -1,4 +1,80 @@
+//generate a number from 1-3
+// 1 = scissors, 2 = stone, 3 = paper
+// programme outputs either scissors, paper or stone
+// if user inputs scissors and programme outputs scissors = draw
+// if user inputs scissors and programme outputs paper = win
+// if user inputs scissors and programme outputs stone = lose
+// if user inputs paper and programme outputs paper = draw
+// if user inputs paper and programme outputs scissors = lose
+// if user inputs paper and programme outputs stone = win
+// if user inputs stone and programme outputs stone = draw
+// if user inputs stone and programme outputs paper = lose
+// if user inputs stone and programme outputs scissors = win
+
+//generating a random number from 0-2
+var randomOutput = function () {
+  var randomNumber = Math.floor(Math.random() * 3);
+  return randomNumber;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  // if the user types in an invalid input
+  //if the computer generates 0, his guess is scissors
+  var randomAttack = randomOutput();
+  if (randomAttack == 0) {
+    var finalAttack = "scissors";
+    console.log(finalAttack);
+  }
+
+  //if the computer generates 1, his guess is stone
+  if (randomAttack == 1) {
+    var finalAttack = "stone";
+    console.log(finalAttack);
+  }
+
+  //if the computer generates 2, his guess is paper
+  if (randomAttack == 2) {
+    var finalAttack = "paper";
+    console.log(finalAttack);
+  }
```

Comment:
Good effort in determining computer's choice. What could be better would be if you could name it clearer and put it in a computerChoice function.